src/app/butter-provider/yts.js

```diff
@@ -4,6 +4,29 @@ const Generic = require('./generic');
 const request = require('request');
 const sanitize = require('butter-sanitize');
 
+const ytsTrackers = [
+  'udp://open.demonii.com:1337/announce',
+  'udp://tracker.openbittorrent.com:80',
+  'udp://tracker.coppersurfer.tk:6969',
+  'udp://glotorrents.pw:6969/announce',
+  'udp://tracker.opentrackr.org:1337/announce',
+  'udp://torrent.gresille.org:80/announce',
+  'udp://p4p.arenabg.com:1337',
+  'udp://tracker.leechers-paradise.org:6969',
+].map(t => `&tr=${t}`).join('');
+
+const ratingsMap = {
+  '1+': '1',
+  '2+': '2',
+  '3+': '3',
+  '4+': '4',
+  '5+': '5',
+  '6+': '6',
+  '7+': '7',
+  '8+': '8',
+  '9+': '9',
+};
```

Comment:
This const can also be removed when config.js rating values are changed back to 'All', '1', '2'..etc and the '+' added in filter-bar.tpl

Reply:
no worries, I've been through worse :) 
I've wrote up the reasoning for doing it this way.
IMO removing them is going to obfuscate this part a bit for maintenance. I strongly believe that "view" components shouldn't mutate the strings that they receive, since it increases the amount of locations to debug. for example I haven't figured out how to debug inside of `.tpl` file to see what goes there

Reply:
I dont think anything mutates. It just formats the label for display purposes by adding a +. Same way __i18 formats it to add the translation. My opinion is that the actual values these labels represent should be the ones that the API expects. Same reason, for simplicity and maintenance, not that your original PR wouldnt work.